app/javascript/.storybook/manager.js

```diff
@@ -0,0 +1,11 @@
+import { addons } from '@storybook/addons';
+import { create } from '@storybook/theming/create';
+
+const crayonsTheme = create({
+  base: 'light',
+  brandTitle: '🖍️🖍️crayons 🖍️🖍️',
```

Comment:
@ludwiczakpawel, it'd be nice to have a crayons logo to pop here instead of my ghetto emoji crayon logo 😉

Reply:
```suggestion
  brandTitle: '🖍️ Crayons 🖍️',
```

using my 24" screen:

**before**

![Screenshot 2020-03-21 at 11 29 27 AM](https://user-images.githubusercontent.com/146201/77224575-6942a400-6b67-11ea-9138-52e52edf870f.png)

**after**

![Screenshot 2020-03-21 at 11 34 21 AM](https://user-images.githubusercontent.com/146201/77224644-f1c14480-6b67-11ea-8ed6-da01ff3d6aa7.png)

Reply:
This is temporary. Waiting for @ludwiczakpawel to whip up a logo.